docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -894,6 +894,28 @@ BUILD(FilterDsLib1);
             compression on the payload. The resulting index can be smaller
             than using lz4.</entry>
           </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4s'</emphasis> </emphasis></entry>
+
+            <entry>Causes inplace compression on the key fields and lz4s
+            compression on the payload. This uses the streaming API to build
+            up a compressed data stream and avoid recompressing it resulting
+            in reduced build times.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4shc'</emphasis> </emphasis></entry>
+
+            <entry>The default compression in versions after versions 9.6.90,
```

Comment:
The default compression for inplace indexes.
Both this and lz4s are only supported in those versions and later.